modules/manage/partials/audit-logging.adoc

```diff
@@ -300,6 +325,16 @@ Some key tuning recommendations for your audit logging settings include:
 * Set `audit_enabled` to `true`.
 * <<Optimize costs for audit logging>>.
 
+The sequence of commands in `rpk` for this audit log configuration is:
+
+ rpk cluster config set audit_log_num_partitions 6
+ rpk cluster config set audit_log_replication_factor 5
+ rpk cluster config set audit_enabled_event_types '["management","describe","authenticate"]'
+ rpk cluster config set audit_excluded_topics '["topic1","topic2"]'
+ rpk cluster config set audit_excluded_principals '["User:principal1", "principal2"]'
+ rpk cluster config set audit_enabled true
+ rpk topic alter-config _redpanda.audit_log --set retention.ms=259200000
+
```

Comment:
These added lines are duplicates of the lines immediately below btw! So it displays twice here https://deploy-preview-1041--redpanda-docs-preview.netlify.app/current/manage/audit-logging/#enable-audit-logging (and not at all for the [Cloud version of the page](https://deploy-preview-1041--redpanda-docs-preview.netlify.app/redpanda-cloud/manage/cluster-maintenance/audit-logging/), please double check that that's correct)

Reply:
fixed